### ISSUE TITLE
Failed to export all accounts for balance history

### ISSUE THREAD

**Issue by tomshaffner** [monarchmoney/mint-export-extension#44]
Worked great for transaction exporting, but when downloading account balances it failed for 30 of 105 accounts. My console in this case shows this:
![Screenshot 2023-12-12 113146](https://github.com/monarchmoney/mint-export-extension/assets/48473431/8ca22053-7e79-4cb8-ba9d-cacb574ab4ee)
Interestingly, in the case of one of my banks the balance history I get includes some of the accounts at it but not all of them, so it doesn't seem to be bank-specific.

A look at the verbose debug screen gives this:
![image](https://github.com/monarchmoney/mint-export-extension/assets/48473431/a7e9e147-3ba9-423d-a4f1-24ea12014c03)

**Comment by idpaterson** [comment]
I suspect that one or two of the improvements in #30 could fix this for you. While my code has not yet been reviewed by the Monarch team, if you are willing to give it a try I have published a prerelease [here](https://github.com/idpaterson/mint-export-extension/releases/latest). If you would like to give it a try, please click that link then under Assets download the chrome-extension.zip file and extract it. Then follow [Monarch's instructions here](https://github.com/monarchmoney/mint-export-extension?tab=readme-ov-file#chrome-web-extension-unpacked) for installing the unpacked extension starting at step 2.

If you decide to give it a try I am interested to hear if it works better!

**Comment by bertique** [comment]
This saved me. I had already migrated to CK and was hitting the rate limit with the latest official version for downloading balances. This downloaded balances for all my xx accounts without an issue. Thank you!

**Comment by idpaterson** [comment]
I'm glad it helped! Interesting result though because all of the bug fixes in that code were submitted separately earlier in January and incorporated into the official extension release version 1.0.8. Do you happen to know whether it was the latest version 1.0.8 on the Chrome web store that was getting rate limited, or could it have been an earlier release? I could have missed something important if 1.0.8 doesn't work for you.

**Comment by bertique** [comment]
Yes, it was on the official 1.0.8 from the web store. Though I just tried again and it was successful with the official app, so not sure what happened earlier. I did get logged out after about 4 accounts and then when logging in, it just showed a page that said something like the service is not available right now. I assume that means I was rate limited, or some other transient issue on the Mint side I assume.
